.claude/skills/cfn-standardized-error-handling/test-error-handling.sh

```
@@ -1,166 +1,166 @@
+# Colored Output
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+# Test Utility Functions
+assert_equals() {
+    local expected="$1"
+    local actual="$2"
+    local test_name="$3"
+
+    if [ "$actual" = "$expected" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)